mirrulations-work-server/tests/test_work_server.py

```diff
@@ -280,3 +280,14 @@ def mock_write_results(mocker):
         'mirrserver.work_server.write_results',
         return_value=None
     )
+
+def test_create_mock_redis_is_connected():
+    redis_server = FakeServer()
+    redis_server.connected = True
+    assert redis_server.connected == True
+
```

Comment:
This assert doesn't test our production code, it tests your first two lines...

I talked with @Glennonr about a technique to accomplish this.  Please connect with him on this.

Reply:
@schumanm I hope the meet is going well!! If you review the pull request I made for the work generator it should have lots of code you can reuse for the work server